ModuleOneSecondTaskLibrary/GreatingBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="GreatingBuilder.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+using System;
+
+namespace ModuleOneSecondTaskLibrary
+{
+    /// <summary>
+    /// Builds greating text.
+    /// </summary>
+    public class GreatingBuilder
+    {
+        /// <summary>
+        /// Returns greating in format:
+        /// «{current_time} Hello, {name}!».
+        /// </summary>
+        /// <returns>Greating text.</returns>
+        public string Build()
```

Comment:
This class and method uses only static information, so they should also be static.